packaging/helm-charts/helm3/Makefile

```diff
@@ -0,0 +1,60 @@
+PROJECT_NAME=helm-charts
+
+include ../../../Makefile.os
+
+RELEASE_VERSION ?= latest
+CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../../../release.version | tr A-Z a-z)
+CHART_NAME=strimzi-drain-cleaner
+CHART_PATH ?= ../helm3/strimzi-drain-cleaner/
+CHART_RENDERED_TEMPLATES_TMP=../../../target/charts
+CHART_RENDERED_TEMPLATES_INSTALL=../../install/certmanager/
+HELM_CLI=helm
+
+helm_clean:
+	rm -rf $(CHART_RENDERED_TEMPLATES_TMP)
+	rm -rf strimzi-$(RELEASE_VERSION)/charts/
+	rm -f $(CHART_NAME)-*.tgz
+
+helm_lint:
+	# Linting Helm Chart
+	$(HELM_CLI) lint --debug $(CHART_PATH)
+
+helm_template:
+	# Generating rendered template files to: $(CHART_RENDERED_TEMPLATES_TMP)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
+	$(HELM_CLI) template strimzi-drain-cleaner --namespace myproject --output-dir $(CHART_RENDERED_TEMPLATES_TMP) $(CHART_PATH)
+
+helm_install: helm_clean helm_template
+	# Remove Helm-related labels
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec $(SED) -i '/^---/d' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec $(SED) -i '/^# Source: /d' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec yq eval -i 'del .metadata.labels."helm.sh/chart"' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec yq eval -i 'del .metadata.labels."app.kubernetes.io/instance"' {}  \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec yq eval -i 'del .metadata.labels."app.kubernetes.io/version"' {} \;
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -exec yq eval -i 'del .metadata.labels."app.kubernetes.io/managed-by"' {} \;
+	# Copying rendered template files to: $(CHART_RENDERED_TEMPLATES_INSTALL)
+	mkdir -p $(CHART_RENDERED_TEMPLATES_INSTALL)
+	# Find rendered resources which are not CustomResourceDefinition and move them
+	$(FIND) $(CHART_RENDERED_TEMPLATES_TMP) -type f -name '*.yaml' -printf "%p " -exec yq eval '.kind' {} \; \
+	| grep -v 'CustomResourceDefinition$$' \
+	| $(SED) -E 's/([^ ]*) ([a-zA-Z0-9]*)$$/\1/' \
+	| xargs -IFILE $(CP) FILE $(CHART_RENDERED_TEMPLATES_INSTALL)
```

Comment:
Is this really needed here given there are no CRDs?

Reply:
This was a poor copy paste effort from the operator project. CRD logic has been removed.